src/App.tsx

```diff
@@ -1,27 +1,55 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
+import { Good } from './types/Good';
+import { getAll, get5First, getRedGoods } from './api/goods';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
+  const handleShowAll = () => {
+    getAll()
+      .then(allGoods => setGoods(allGoods));
```

Comment:
You write kind of the same code 3 times. Better create a helper function that would receive `callback` and `setGoods` for needed `goods`.
And then call it for each `onClick` with the needed callback function as a parameter.